request-controller/src/main/java/org/wildfly/extension/requestcontroller/RequestController.java

```diff
@@ -59,7 +59,7 @@
  */
 public class RequestController implements Service<RequestController>, ServerActivity {
 
-    public static final ServiceName SERVICE_NAME = ServiceName.JBOSS.append("server", "global-request-controller");
+    public static final ServiceName SERVICE_NAME = RequestControllerRootDefinition.REQUEST_CONTROLLER_CAPABILITY.getCapabilityServiceName();
```

Comment:
I think we should keep the old name for now, maybe deprecate it, and set it as an alias on the service.

The service should register the name with `OperationContext.getCapabilityServiceName("org.wildfly.request-controller", RequestController.class)`.